templates/download/server/index.html

```diff
@@ -5,100 +5,64 @@
-          <li class="p-list__item is-ticked">Supports Windows, Ubuntu, CentOS and RHEL</li>
-        </ul>
-        <p>
-          <a href="https://maas.io" class="p-link--external">Get MAAS</a>
-        </p>
-      </div>
+      <p class="u-no-padding--top">
+        The long-term support version of Ubuntu Server, including the {{ releases.openstack_lts.slug }} release of OpenStack and support guaranteed until {{ releases.lts.eol }} &mdash; 64-bit only.
+      </p>
+      <p>
+        This release uses our new installer, Subiquity. If you need support for options not implemented in Subiquity, such as encrypted filesystem support, the traditional installer can be found on the <a href="/download/alternative-downloads#alternate-ubuntu-server-installer">alternative downloads</a> page.
+      </p>
+      <p>
+        <a href="https://wiki.ubuntu.com/{{ releases.lts.slug }}/ReleaseNotes" class="p-link--external">Ubuntu Server {{ releases.lts.short_version }} LTS release notes</a>
+      </p>
+    </div>
+    <div class="col-4">
+      <p>
+        <a href="/download/server/thank-you?version={{ releases.lts.full_version }}&amp;architecture=amd64" class="p-button--positive is-wide" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Server', 'eventLabel' : '{{ releases.lts.short_version }} LTS', 'eventValue' : undefined });">Download</a>
+      </p>
+      <p>
+        <small>For other versions of Ubuntu including torrents, the network installer, a list of local mirrors, and past releases <a href="/download/alternative-downloads">see our alternative downloads</a>.</small>
+      </p>
     </div>
   </div>
-</section>
-
-<section class="p-strip is-bordered is-shallow">
-  <div class="row">
-    <div class="col-12">
-      <h2>Ubuntu Server {{ releases.lts.full_version }} <abbr title="Long-term support">LTS</abbr> for OpenStack</h2>
-      <ul class="p-list is-split">
-        <li class="p-list__item is-ticked">Supported for five years by Canonical</li>
-        <li class="p-list__item is-ticked">Updated to the OpenStack {{ releases.openstack_lts.slug }} release, including Designate, more flexible Ceph deployments and improved monitoring.</li>
-        <li class="p-list__item is-ticked">Nova LXD driver - deploy OpenStack instances as system containers, dramatically increasing per-node density</li>
-        <li class="p-list__item is-ticked">Juju OpenStack bundle -  automated deployment of OpenStack into LXD system containers</li>
-        <li class="p-list__item is-ticked">ZFS support for LXD OpenStack hosts</li>
-        <li class="p-list__item is-ticked">Certified by Microsoft to host Windows Server 2012 and Windows Server 2008 R2 as guests, under its Server Virtualisation Validation Program (SVVP)</li>
-        <li class="p-list__item is-ticked">Multi platform support OpenStack, Juju, MAAS, LXD on IBM LinuxONE mainframe, IBM Power 8 and ARM64</li>
-      </ul>
+  <div class="u-fixed-width">
```

Comment:
I think that you can get a similar effect without hard-coding a style.

Something like `<div class="p-strip is-narrow"><hr></div>`